SOPs/toil-rnaseq-unc.sop.md

```diff
@@ -0,0 +1,90 @@
+Steps to run the RNA-seq UNC pipeline vs the RNA-seq CGL pipeline
+
+Setup the cluster the same way, but instead of needing a venv and pip installing toil-scripts, 
+`git clone toil-scripts -b releases/2.0.x`
+you'll also need to use `cgcloud ssh-cluster` to `pip uninstall toil` and reinstall `pip install toil==3.1.6` to all the nodes
+ 
+There is a convenience script for running on mesos, which you'll edit to change bucket other settings etc.  
+The config you create (there is an example in the directory), must be on every machine, so you'll use the `--share` 
+option and pass in a location where that file is, which will be on `/home/mesosbox/shared`
+You will also need to have the master key file in the shared directory
+
+1) Setup CGCloud
+See the SOP https://github.com/BD2KGenomics/core-operations/blob/master/SOPs/toil-rnaseq.sop.md
```

Comment:
A relative link would be better. If I were looking at this file in a branch, an absolute link would lead me to the file on master. And that's typically not what you'd want.